### ISSUE TITLE
E127: pycodestyle shows E127 while code seems to be formatted correctly

### ISSUE THREAD

**Issue by stanislawskwark** [PyCQA/pycodestyle#862]
The following code results with pycodestyle warning 
"E127 continuation line over-indented for visual indent"
```
for i in range(5):
    print("this_line_causes_the_warning")
    assert i == \
           3
```

while for following pieces of code there are no warnings:
```
for i in range(5):
    assert i == \
           3
```

```
print()
assert 3 == \
       3, "some error"
```
pycodestyle version - 2.5.0
Python version - 3.7.2

**Comment by FichteFoll** [comment]
Confirmed on 2.5.0 and 3.7.3.

**Comment by FichteFoll** [comment]
Confirmed on 2.5.0 and 3.7.3.

**Comment by asottile** [comment]
this appears fixed in pycodestyle 2.6.0 -- I believe by #912
